### PR TITLE
fix(plasma-web,plasma-b2c): fix various api diff between web/b2c

### PR DESCRIPTION
**Pull request by Yeti-or**
Разобрали оставшиеся кусочки plasma-web/plasma-b2c которые создавала различия в API библиотек, где-то что-то забыли, в основном мелочковка

:baby_chick: Download canary assets:

[colors--canary.458.4598067088.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.458.4598067088.xml)  
[color_sberHealth_ios-swift--canary.458.4598067088.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.458.4598067088.swift)  
[color_sberHealth_kotlin--canary.458.4598067088.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.458.4598067088.kt)  
[color_sberHealth_react-native--canary.458.4598067088.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.458.4598067088.ts)  
[color_sbermarket_business_ios-swift--canary.458.4598067088.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.458.4598067088.swift)  
[color_sbermarket_business_kotlin--canary.458.4598067088.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.458.4598067088.kt)  
[color_sbermarket_business_react-native--canary.458.4598067088.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.458.4598067088.ts)  
[color_sbermarket_ios-swift--canary.458.4598067088.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.458.4598067088.swift)  
[color_sbermarket_kotlin--canary.458.4598067088.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.458.4598067088.kt)  
[color_sbermarket_metro_ios-swift--canary.458.4598067088.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.458.4598067088.swift)  
[color_sbermarket_metro_kotlin--canary.458.4598067088.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.458.4598067088.kt)  
[color_sbermarket_metro_react-native--canary.458.4598067088.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.458.4598067088.ts)  
[color_sbermarket_react-native--canary.458.4598067088.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.458.4598067088.ts)  
[color_sbermarket_selgros_ios-swift--canary.458.4598067088.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.458.4598067088.swift)  
[color_sbermarket_selgros_kotlin--canary.458.4598067088.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.458.4598067088.kt)  
[color_sbermarket_selgros_react-native--canary.458.4598067088.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.458.4598067088.ts)  
[color_sbermarket_vinlab_ios-swift--canary.458.4598067088.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_vinlab_ios-swift--canary.458.4598067088.swift)  
[color_sbermarket_vinlab_kotlin--canary.458.4598067088.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_vinlab_kotlin--canary.458.4598067088.kt)  
[color_sbermarket_vinlab_react-native--canary.458.4598067088.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_vinlab_react-native--canary.458.4598067088.ts)  
[color_sberprime_ios-swift--canary.458.4598067088.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.458.4598067088.swift)  
[color_sberprime_kotlin--canary.458.4598067088.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.458.4598067088.kt)  
[color_sberprime_react-native--canary.458.4598067088.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.458.4598067088.ts)  
[PlasmaTokensColor--canary.458.4598067088.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.458.4598067088.swift)  
[shadow_sbermarket_react-native--canary.458.4598067088.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.458.4598067088.ts)  
[typo_mage_ios-swift--canary.458.4598067088.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.458.4598067088.swift)  
[typo_mage_kotlin--canary.458.4598067088.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.458.4598067088.kt)  
[typo_mage_react-native--canary.458.4598067088.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.458.4598067088.ts)  
[typo_plasma_ios-swift--canary.458.4598067088.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.458.4598067088.swift)  
[typo_plasma_kotlin--canary.458.4598067088.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.458.4598067088.kt)  
[typo_plasma_react-native--canary.458.4598067088.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.458.4598067088.ts)  
[typo_ruler_ios-swift--canary.458.4598067088.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.458.4598067088.swift)  
[typo_ruler_kotlin--canary.458.4598067088.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.458.4598067088.kt)  
[typo_ruler_react-native--canary.458.4598067088.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.458.4598067088.ts)  
[typo_sage_ios-swift--canary.458.4598067088.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.458.4598067088.swift)  
[typo_sage_kotlin--canary.458.4598067088.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.458.4598067088.kt)  
[typo_sage_react-native--canary.458.4598067088.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.458.4598067088.ts)  
[typo_sbermarket_ios-swift--canary.458.4598067088.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.458.4598067088.swift)  
[typo_sbermarket_kotlin--canary.458.4598067088.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.458.4598067088.kt)  
[typo_sbermarket_react-native--canary.458.4598067088.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.458.4598067088.ts)  
[typo_soulmate_ios-swift--canary.458.4598067088.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.458.4598067088.swift)  
[typo_soulmate_kotlin--canary.458.4598067088.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.458.4598067088.kt)  
[typo_soulmate_react-native--canary.458.4598067088.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.458.4598067088.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.173.1-canary.458.4598067088.0
  npm install @salutejs/plasma-hope@0.31.1-canary.458.4598067088.0
  npm install @salutejs/plasma-temple@1.148.1-canary.458.4598067088.0
  npm install @salutejs/plasma-ui@1.180.1-canary.458.4598067088.0
  npm install @salutejs/plasma-web@1.198.1-canary.458.4598067088.0
  # or 
  yarn add @salutejs/plasma-b2c@1.173.1-canary.458.4598067088.0
  yarn add @salutejs/plasma-hope@0.31.1-canary.458.4598067088.0
  yarn add @salutejs/plasma-temple@1.148.1-canary.458.4598067088.0
  yarn add @salutejs/plasma-ui@1.180.1-canary.458.4598067088.0
  yarn add @salutejs/plasma-web@1.198.1-canary.458.4598067088.0
  ```
</details>

### CODE REVIEW

File: reports/plasma-hope-diff.diff [hunk 1]
```diff
@@ -6,556 +6,17 @@
  
  > Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
```
Comment (Yeti-or) [inline reports/plasma-hope-diff.diff:7]: @Yakutoc @neretin-trike зацените итоговый diff

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-458/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-458/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-458/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-458/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-458/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-458/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-458/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

File: reports/plasma-hope-diff.diff (hunk 1 above)
Comment (neretin-trike) [reply reports/plasma-hope-diff.diff:7]: опиши пожалуста, что сделал в прке и для чего?

Comment (neretin-trike) [reply reports/plasma-hope-diff.diff:7]: а дифф выглядит супер классно

**Review by Yakutoc — APPROVED** [review]

**Review by neretin-trike — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-458/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-458/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-458/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-458/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-458/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-458/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Review by Yakutoc — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-458/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-458/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-458/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-458/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-458/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-458/temple-storybook/

**Pull request merged**